### ISSUE TITLE
PDF/UA 7.4.2 test 1 and WCAG 4.1.2 tests 9, 10 possible false positives

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#72]
@bdoubrov , [this PDF document](https://forumstandaardisatie.nl/sites/default/files/FS/2021/0609/FS-20210609.3A-Forumadvies-functioneel-toepassingsgebied-WPA2-Enterprise.pdf) generates a number of errors that appear to be false positives:

- **PDF/UA 7.4.2 test 1 "Incorrect use of heading levels"** points to a line that is tagged as heading 2. The document however does appear to have a heading 1 that precedes it (the first lines of the document just below the logo were formatted as a heading 1 in the Word source document). So the use of heading levels appears to be correct.
- **WCAG 4.1.2 test 9 "Missing table elements"** points to text all over the document. This test appears to mistake normal text for tables.
- **WCAG 4.1.2 test 10 "Heading is incorrectly tagged as a paragraph"** points to three different lines of text. The first was formatted in the Word source document as a heading 1, so this may be a false negative.  The second points to a line that was formatted as a subtitle; in this case the algorithm may be right in thinking this was intended as a heading. The third points to a paragraph set in boldface with borders; here it's less clear why the algorithm thinks this was intended as a heading.

**Comment by bdoubrov** [comment]
Heading errors are related to the fact that the document has no H1 header and the very first line of text (notitie) is not tagged as a header.

Some table errors (but not all!) show text structures, that might be visually confused with tables. We'll check if we can report less false positives in this document.

**Comment by jzuidweg** [comment]
You're right, this document does not appear to have a H1 heading. What's curious is that in the Word source the first lines were formatted as H1, but apparently this was lost in the PDF export. I'll have to check why this happens. 

But in any case the checker came to the right conclusion so PDF/UA 7.4.2 test 1 does not give a false positive here.

**Comment by jzuidweg** [comment]
This issue has been re-tested and solved. It can be closed.

**Comment by jzuidweg** [comment]
This issue is marked as fixed-in-dev but **WCAG 4.1.2 test 9** "_Missing table elements_" and **WCAG 4.1.2 test 10** "_Heading is incorrectly tagged as a paragraph_" continue giving false positives. I'm not closing this issue yet.

**Comment by bdoubrov** [comment]
After the latest dev update we still get these two errors, but I think for a good reason: 
- WCAG 4.1.2 test 9 "Missing table elements": The reported missing table covers the information on the 1st page, which indeed looks like a table, but is not tagged as such
- WCAG 4.1.2 test 10 "Heading is incorrectly tagged as a paragraph" is now reported only about a paragraph set in boldface with borders on the 1st page, which indeed seems to be a header, but is using <P> tag instead
